app/__tests__/components/JsonOutputModal.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import { JsonOutputModal, JsonOutputModalProps } from "../../components/JsonOutputModal";
+import { ensStampFixture } from "../../__test-fixtures__/databaseStorageFixtures";
+
+let props: JsonOutputModalProps;
+const passport = {
+  issuanceDate: "2022-06-14T08:51:28.157Z",
+  expiryDate: "2022-06-14T08:51:28.157Z",
+  stamps: [],
+};
+
+beforeEach(() => {
+  props = {
+    isOpen: true,
+    onClose: jest.fn(),
+    subheading: "Custom subheading",
+    title: "Modal Title",
+    jsonOutput: passport,
+    closeButtonText: "Custom Close",
+  };
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("displays the modal", () => {
+  it("should not display your modals content", () => {
```

Comment:
```suggestion
  it("should display your modals content", () => {
```
This test is checking for the content to be displayed. The title should indicate that.